### PR TITLE
chore: localize swap/buy errors.

### PR DESCRIPTION
**Pull request by rcresnik**

### CODE REVIEW

File: brd-ios/breadwallet/src/Constants/Strings.swift [hunk 1]
```diff
@@ -687,16 +701,48 @@ internal enum L10n {
...
     internal static let noCurrencies = L10n.tr("Localizable", "ErrorMessages.NoCurrencies")
+    /// Failed to fetch network fees. Please try again later.
+    internal static let noFees = L10n.tr("Localizable", "ErrorMessages.noFees")
+    /// No quote for currency pair %s-%s.
+    internal static func noQuoteForPair(_ p1: UnsafePointer<CChar>, _ p2: UnsafePointer<CChar>) -> String {
+      return L10n.tr("Localizable", "ErrorMessages.noQuoteForPair", p1, p2)
+    }
+    /// Please make sure you have enough ETH to cover for the network fees while swapping within Ethereum-based assets.
+    internal static let notEnoughEthForFee = L10n.tr("Localizable", "ErrorMessages.notEnoughEthForFee")
+    /// The amount is higher than your daily limit of $1,000 USD. Please upgrade your account or enter a lower amount.
```
Comment (kenan-mamedoff) [inline brd-ios/breadwallet/src/Constants/Strings.swift]: hardcoded values?

File: brd-ios/breadwallet/src/Constants/Strings.swift [hunk 2]
```diff
@@ -687,16 +701,48 @@ internal enum L10n {
...
+    internal static let overDailyLimit = L10n.tr("Localizable", "ErrorMessages.overDailyLimit")
+    /// Over exchange limit.
+    internal static let overExchangeLimit = L10n.tr("Localizable", "ErrorMessages.overExchangeLimit")
+    /// The amount is higher than your lifetime limit of $10,000 USD. Please upgrade your account or enter a lower amount.
```
Comment (kenan-mamedoff) [inline brd-ios/breadwallet/src/Constants/Strings.swift]: hardcoded values?

File: brd-ios/breadwallet/src/Constants/Strings.swift [hunk 3]
```diff
@@ -687,16 +701,48 @@ internal enum L10n {
...
+    internal static let overLifetimeLimit = L10n.tr("Localizable", "ErrorMessages.overLifetimeLimit")
+    /// The amount is higher than your daily limit of $10,000 USD. Please enter a lower amount.
```
Comment (kenan-mamedoff) [inline brd-ios/breadwallet/src/Constants/Strings.swift]: hardcoded values?

File: brd-ios/breadwallet/src/Constants/Strings.swift [hunk 4]
```diff
@@ -687,16 +701,48 @@ internal enum L10n {
...
+    internal static let overLifetimeLimitLevel2 = L10n.tr("Localizable", "ErrorMessages.overLifetimeLimitLevel2")
+    /// A maximum of one swap can be active for a currency at a time.
+    internal static let pendingExchange = L10n.tr("Localizable", "ErrorMessages.pendingExchange")
+    /// PIN Authentication failed.
+    internal static let pinConfirmationFailed = L10n.tr("Localizable", "ErrorMessages.pinConfirmationFailed")
+    /// In order to succesfully perform a swap, make sure you have two or more of our supported swap assets (BSV, BTC, ETH, BCH, SHIB, USDT) activated and funded within your wallet.
```
Comment (kenan-mamedoff) [inline brd-ios/breadwallet/src/Constants/Strings.swift:744]: hardcoded values?

File: brd-ios/breadwallet/src/Redesign/Scenes/Swap/Main/SwapErrors.swift [hunk 5]
```diff
@@ -31,59 +31,55 @@ enum SwapErrors: FEError {
...
             
         case .networkFee:
-            return "This swap doesn't cover the included network fee. Please add more funds to your wallet or change the amount you're swapping."
+            return L10n.ErrorMessages.networkFee
             
-        case .noQuote(let pair):
-            return "No quote for currency pair \(pair ?? "<missing>")."
+        case .noQuote(let from, let to):
+            let from = from ?? "/"
+            let to = to ?? "/|"
```
Comment (kenan-mamedoff) [inline brd-ios/breadwallet/src/Redesign/Scenes/Swap/Main/SwapErrors.swift]: "/"

File: brd-ios/breadwallet/src/Constants/Strings.swift (hunk 1 above)
Comment (rcresnik) [reply brd-ios/breadwallet/src/Constants/Strings.swift]: job was localization.. :D

File: brd-ios/breadwallet/src/Redesign/Scenes/Swap/Main/SwapErrors.swift (hunk 5 above)
Comment (rcresnik) [reply brd-ios/breadwallet/src/Redesign/Scenes/Swap/Main/SwapErrors.swift]: what would u replace an optional war with?

Comment (kenan-mamedoff) [reply brd-ios/breadwallet/src/Redesign/Scenes/Swap/Main/SwapErrors.swift]: just saying there is a typo. it should be "/" not "/|"

**Review by kenan-mamedoff — APPROVED** [review]

**Pull request merged**